server/dataGeneration/generateCopyAuthors.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs');
+const path = require('path');
+
+const query = 'COPY public.authors (id, first_name, last_name, email, password) FROM stdin;';
```

Comment:
Are these authors copied to access Postgres? I don't know much about Postgres at the moment, but this looks good to me.

Reply:
Yes, in the final `dump.sql` file there are millions of rows just under this line with the values for each column to be copied in